src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -484,7 +484,7 @@ public String invoke(RevisionInternal rev) {
         body.put("keys", keys);
 
         Future future = sendAsyncRequest("POST",
-                "/_all_docs?include_docs=true",
+                "_all_docs?include_docs=true",
```

Comment:
With "/", Generated URL should be absolute path <host>/<path>.
Without "/", Generated URL should be relative path <host>/<db>/<path>

CBL Android/Java had issue in URL generation codes which is not compatible with iOS implementation. So I fixed this.